api/v1alpha/src/lib/config.js

```diff
@@ -34,11 +34,11 @@ config.cdsAccountViewFields = new Set(['rowId', 'accountId', 'email', 'emailType
     'accountType', 'createdBy', 'policies', 'marketplace', 'createdAt', 'version', 'isDeleted']);
 
 config.cdsPolicyTableId = "policy";
-config.cdsPolicyTableFields = new Set(['rowId', 'policyId', 'name', 'description',
+config.cdsPolicyTableFields = new Set(['rowId', 'policyId', 'name', 'description', 'isTableBased',
```

Comment:
Do we need an extra flag in the schema or can table policy logic be determined inherently from the dataset data object? i.e. `datasets.name.tables == isTableBased` vs `datasets.name == isNotTableBased` ?

Reply:
I thought about that, but I also considered it would be a nice-to-have to maintain the list of tables selected, even if a user decided to toggle back to dataset-based and then potentially back to table-based. The flag will indicate which to use, while we retain the data for tables which may not be used.

Reply:
Ok. Gotcha.